client/utils/consts.js

```diff
@@ -1,4 +1,5 @@
 export default Object.freeze({
+  SHOW_EXAMPLES: true, // Allow /examples
```

Comment:
This is how we can enable/diable example menu, and routes.

Reply:
👍